visual-testing-app/src/components/theme-switcher/theme-switcher.tsx

```diff
@@ -0,0 +1,40 @@
+import { themes, designTokens } from '@commercetools-uikit/design-system';
+
+export type ThemeName = keyof typeof themes;
+
+type ThemeSwitcherProps = {
+  selectedTheme: string;
+  onThemeChange(theme: ThemeName): void;
+};
+
+const ThemeSwitcher = ({
```

Comment:
This is a basic utility I used during development to be able to compare default and new theme.
I guess it's helpful but we can remove it if we don't want it (or make it looks nicer if we want to keep it).

Reply:
FYI: we might want to add it in the AppShell as well, making it visible behind the feature flag. Then we can toggle the designs manually instead of restricting the usage to certain projects.

cc @tdeekens As he was asking about this.

Reply:
Sure.
I'll do that in a follow-up PR.